app/src/main/res/layout/add_edit_activity.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/imageView"
```

Comment:
need a more specific id here

Reply:
It does not make sense to me that whenever I change imageView to something else (i.e book_image_view), the UI is messed up.

Reply:
try using refactor -> rename (in right click menu) instead of changing it directly

Reply:
Yes that works